docs/en_US/guide/sources/builtin/websocket.md

```diff
@@ -0,0 +1,83 @@
+# Websocket Source Connector
+
+<span style="background:green;color:white;">stream source</span>
+
+eKuiper has built-in support for Websocket data sources. Through the Websocket data source connector, eKuiper can obtain data from external Websocket clients.
+
+## Server Configuration
+
+To set up eKuiper as an Websocket endpoint, configure the server settings in `etc/sources/websocket.yaml`.
+
+```yaml
+source:
+  ## Configurations for the global websocket server for websocket source
+  # HTTP data service ip
+  httpServerIp: 0.0.0.0
```

Comment:
In the implementation, I see you use the same server as httppush. If so, the server ip/port is a global configuration in kuiper.yaml, not in the source side

Reply:
Path can use dataSource to configure